params/config.go

```diff
@@ -504,6 +504,12 @@ type Network struct {
 	Enabled                bool   `json:"enabled"`
 }
 
+type CommunitySettings struct {
+  CommunityID                     string          `json:"communityId"`
+  MessageArchiveFetchingEnabled   bool            `json:"messageArchiveFetchingEnabled"`
+  MessageArchiveSeedingEnabled    bool            `json:"messageArchiveSeedingEnabled"`
+}
```

Comment:
This might not be the proper place to put this type. 

Eventually, we probably need a type for enabling/disabling community history archive support in the CLI as well. Might as well leverage this type for that, which is why I put it here.

Happy to put it somewhere else.